src/pages/projects/actions/DeleteProjectBtn.tsx

```diff
@@ -10,15 +10,70 @@ import { useSmallScreen } from "context/useSmallScreen";
 import {
   ConfirmationButton,
   Icon,
+  Tooltip,
   useNotify,
 } from "@canonical/react-components";
 import classnames from "classnames";
 import { useToastNotification } from "context/toastNotificationProvider";
+import { filterUsedByType } from "util/usedBy";
+import { ResourceType } from "util/resourceDetails";
 
 interface Props {
   project: LxdProject;
 }
 
+const generateProjectUsedByTooltip = (project: LxdProject) => {
+  const resourceLabelAndLink: Record<string, { label: string; link: string }> =
```

Comment:
```suggestion
  const resourceLabelAndLink: Record<ResourceType, { label: string; link: string }> =
```

Reply:
chose `string` here as I didn't want to specify a key for `snapshot`, also later when iterating the keys in `resourceLabelAndLink` we would still need to cast those keys from `string` to `ResourceType`?

Reply:
Ah right, thought we could streamline this.